services/airflow/src/lib/dag_helpers/test_write_dataframe_to_postgres.py

```diff
@@ -71,6 +72,40 @@ def test_insert_dataframe_to_postgres_does_not_overwrite(db_context: DBContext):
     assert res[0] == (0, "DE", "Deutschland", 0)
 
 
+def test_truncate_insert_dataframe_to_postgres_does_replace_existing_data(
+    db_context: DBContext,
+):
+    connect_to_db_and_insert_pandas_dataframe(
+        schema="coronacases", table="german_counties_more_info", data=DATA4
+    )
+    connect_to_db_and_truncate_insert_pandas_dataframe(
+        schema="coronacases", table="german_counties_more_info", data=DATA4
+    )
```

Comment:
Write different data the second time and test that only data of second write persists

Reply:
done